packages/server/dataloader/hydrateRedisDoc.ts

```diff
@@ -1,15 +1,13 @@
-import IUser from '../postgres/types/IUser'
-
 const hydrators = {
   publicTemplates: (publicTemplates: any[]) => {
     publicTemplates.forEach((template) => {
       template.createdAt = new Date(template.createdAt)
     })
     return publicTemplates
   },
-  User: (user: IUser) => {
+  User: (user: any) => {
```

Comment:
+1 i think this hydrator is safe to remove now that we're always fetching users from PG, but happy to treat that as out of scope